src/symmetry.jl

```diff
@@ -200,13 +207,31 @@ function accumulate_over_symops!(ρaccu, ρin, basis, symops, Gs)
     ρaccu
 end
 
+# Low-pass filters ρ (in Fourier) so that symmetry operations acting on it stay in the grid
+function lowpass_for_symmetry!(ρ, basis, symops=basis.symops)
```

Comment:
It's kind of internal, since it is specific to symmetrize. I'd add an underscore or so.

Reply:
I agree it's internal, but self-contained enough to be called outside of symmetrize. It's not more internal than many of other functions we have and we don't really put underscores anywhere else. Internal is evidenced by the fact it has a comment, not a docstring.